src/org/ods/component/ScanWithAquaStage.groovy

```diff
@@ -18,7 +18,7 @@ class ScanWithAquaStage extends Stage {
     static final String CRITICAL_AQUA_SEVERITY = 'critical'
     static final String STAGE_NAME = 'Aqua Security Scan'
     static final String AQUA_CONFIG_MAP_NAME = "aqua"
-    static final String BITBUCKET_AQUA_REPORT_KEY = "org.opendevstack.aquasec"
+    static final String BITBUCKET_AQUA_REPORT_KEY = "ods.sec"
```

Comment:
Is this change needed? If needed, it should be synchronized with the key in ScanWithSonarStage

Reply:
this is to cater to the natural 50 char limit for the key in bitbucket api ..